apps/api/src/controllers/competition.controller.ts

```diff
@@ -384,14 +384,7 @@ export function makeCompetitionController(services: ServiceRegistry) {
           queryParams,
         });
 
-        res.status(200).json({
-          ...result,
-          pagination: buildPaginationResponse(
-            result.total,
-            queryParams.limit,
-            queryParams.offset,
-          ),
-        });
```

Comment:
Moved the building of the pagination object into the service

Reply:
interesting, I've mostly been going the other direction in my migrations, since response format structure feels like more the concern of the handler than the service.  But its pretty minor, I don't feel strongly either way